common/src/main/java/com/gregtechceu/gtceu/common/data/GTMachines.java

```diff
@@ -484,13 +484,13 @@ public static BiConsumer<ItemStack, List<Component>> createTankTooltips(String n
     //////////////////////////////////////
     //*******     Multiblock     *******//
     //////////////////////////////////////
-    public final static MultiblockMachineDefinition LARGE_BOILER_BRONZE = registerLargeBoiler("large_bronze_boiler", CASING_BRONZE_BRICKS, CASING_BRONZE_PIPE, FIREBOX_BRONZE,
+    public final static MultiblockMachineDefinition LARGE_BOILER_BRONZE = registerLargeBoiler("Bronze", CASING_BRONZE_BRICKS, CASING_BRONZE_PIPE, FIREBOX_BRONZE,
```

Comment:
Because of how I changed the registerLargeBoiler function from below.
They take in a singular material, this comes a part of the ingame name.
the .ToLowercase() appears in the ID.